packages/bits/src/services/unit-conversion.service.ts

```diff
@@ -38,20 +36,13 @@ export class UnitConversionService {
                 this.logger.warn("unit conversion service does not support conversion to negative order of magnitude");
             }
 
-            // fix the precision edge case
-            const valueCeiled = Math.ceil(resultValue);
-            if (valueCeiled % base === 0) {
-                resultValue = valueCeiled / base;
-                resultOrder += 1;
-            }
```

Comment:
There was a bug where values approaching conversion to the next higher unit would be prematurely converted. Removing this fixes that.

I pinpointed the origin of this block of code [here](https://github.com/solarwinds/orion-nova-novajs/commit/ecd43c058b50af9777f3e309b3d632e9efa9afac#diff-32871140f4e96e556288e082303b67f32901f119c54bd3ebb9d148d668afc248), and we've maintained the same [unit tests](https://github.com/solarwinds/orion-nova-novajs/commit/ecd43c058b50af9777f3e309b3d632e9efa9afac#diff-14de2330bc8407bdb155138ca489f254d72ef401d878d69fea889cc134f0d40f) that were created in the same PR as this code's origin.

All of our current unit tests pass, and I'm unable to detect what the "edge case" referred to in the comment might have been.